clients/adapter/src/main/java/org/eclipse/hono/adapter/client/telemetry/EventSender.java

```diff
@@ -47,7 +47,7 @@
      *         The future will be failed with a {@code org.eclipse.hono.client.ServerErrorException} if the data
      *         could not be sent. The error code contained in the exception indicates the
      *         cause of the failure.
-     * @throws NullPointerException if tenant ID, device ID or contentType are {@code null}.
+     * @throws NullPointerException if tenant ID or device ID are {@code null}.
```

Comment:
how about adding a note to the *contentType* param that it will be set to the fallback content-type if a payload is given but no content-type is specified?

Reply:
Yes. I'll do that in the next commit (see below), then aligned with the updated javadoc for `MessageHelper#newMessage`.